vocabularies/qld-mining-permits.ttl

```diff
@@ -0,0 +1,305 @@
+
+Infrastructure development - petroleum facility licence and petroleum pipeline licence 
+
+Information gathering - petroleum survey licence, water monitoring authority and data acquisition authority."""@en ;
+    skos:member minpermit:AuthorityToProspect,
+        minpermit:DataAcquisitionAuthority,
+        minpermit:PetroleumFacilityLicence,
+        minpermit:PetroleumLease,
+        minpermit:PetroleumPipelineLicence,
+        minpermit:PetroleumSurveyLicence,
+        minpermit:PotentialCommercialArea,
+        minpermit:WaterMonitoringAuthority ;
+    skos:prefLabel "Petroleum and gas resource authority"@en .
+
+minpermit:ProspectingPermit a skos:Concept ;
+    skos:definition """A prospecting permit can be sought for any mineral other than coal. There are 2 types of prospecting permit:
+
+A prospecting permit allows you to prospect, hand-mine and peg a mining lease or mining claim (if marking out is required) on the available land specified in the permit.
+
+Hand-mining means using hand-operated tools, such as jackhammers, picks, shovels, gads, sieves and windlasses."""@en ;
+    skos:inScheme minpermit:conceptScheme ;
+    skos:notation "PP" ;
+    skos:prefLabel "Prospecting Permit"@en ;
+    skos:topConceptOf minpermit:conceptScheme .
+
+minpermit:AuthorityToProspect a skos:Concept ;
+    skos:altLabel "Exploration Permits for Petroleum (EPP)"@en ;
+    skos:definition """To explore for petroleum, oil, coal seam gas and natural gas in Queensland, you must hold a current authority to prospect (ATP).
+
+An ATP allows you to:
+
+Explore for petroleum
+
+Test for petroleum production
+
+Evaluate the feasibility of petroleum production
+
+Evaluate or test natural underground reservoirs for the storage of petroleum or a prescribed storage gas."""@en ;
+    skos:inScheme minpermit:conceptScheme ;
+    skos:notation "ATP" ;
+    skos:prefLabel "Authority to Prospect"@en ;
+    skos:topConceptOf minpermit:conceptScheme .
+
+minpermit:ExplorationPermitCoal a skos:Concept ;
+    skos:definition "Exploration permit for coal allows you to use more advanced exploration methods to determine the quantity and quality of coal present."@en ;
+    skos:inScheme minpermit:conceptScheme ;
+    skos:notation "EPC" ;
+    skos:prefLabel "Exploration Permit Coal"@en ;
+    skos:topConceptOf minpermit:conceptScheme .
+
+minpermit:ExplorationPermitGeothermal a skos:Concept ;
+    skos:definition "An EPG allows you to explore for geothermal resources and evaluate the feasibility of geothermal production, including by production testing."@en ;
+    skos:inScheme minpermit:conceptScheme ;
+    skos:notation "EPG" ;
+    skos:prefLabel "Exploration Permit Geothermal"@en ;
+    skos:topConceptOf minpermit:conceptScheme .
+
+minpermit:ExplorationPermitGreenhouseGas a skos:Concept ;
+    skos:definition """To search for greenhouse gas (GHG) capture and storage locations in Queensland you need an exploration permit for greenhouse gas (EPQ).
```

Comment:
triple quote marks at start and end of definition. Occurs throughout vocabulary. Needs bulk find and replace

Reply:
https://www.w3.org/TR/sparql11-query/#QSynLiterals

To facilitate writing literal values which themselves contain quotation marks or which are long and contain newline characters, SPARQL provides an additional quoting construct in which literals are enclosed in three single- or double-quotation marks.

The lines affected contain the newline character \n